igor-web/src/main/groovy/com/netflix/spinnaker/igor/Main.groovy

```diff
@@ -48,6 +48,8 @@ class Main extends SpringBootServletInitializer {
     ]
 
     static {
+        System.setProperty("spring.main.allow-bean-definition-overriding", "true");
```

Comment:
I don't think this is necessary anymore - I pulled this out into kork.

Reply:
good call, saw that in kork